cl/search/models.py

```diff
@@ -320,6 +324,13 @@ class Docket(AbstractDateTimeModel):
         (SCRAPER_AND_HARVARD, "Scraper and Harvard"),
         (RECAP_AND_SCRAPER_AND_HARVARD, "RECAP, Scraper and Harvard"),
         (HARVARD_AND_COLUMBIA, "Harvard and Columbia"),
+        (IDB_AND_HARVARD, "IDB and Columbia"),
```

Comment:
Is this correct?

Reply:
No, my mistake, "IDB and Harvard" is the correct option name